test/app/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@okta/okta-signin-widget": "link:../../dist"
```

Comment:
@denysoblohin-okta you had originally added this for reasons related to the ESM build for the test app. @trevoring-okta and I couldn't find any problems running the ESM build now with this change, so maybe it's not longer a problem now. Can you take a look and explain if there's any cases that you think will fail?

Reply:
@shawyu-okta Sorry forgot to update you directly via Slack DM but I already checked with @denysoblohin-okta and he verified that the `link:../../dist` is no longer necessary

Reply:
Ok nice, no worries